api/src/test/resources/org/openmrs/api/include/OrderServiceTest-createOrderGroup.xml

```diff
@@ -17,4 +17,15 @@
 <order_group order_group_id="3" order_set_id="2000" patient_id="7" encounter_id="3"   creator="1" date_created="2012-02-19 12:24:10.0" voided="false" uuid="1c96f25c-4949-4f72-9931-d808fbcdb425" />
 <order_group order_group_id="4" order_set_id="2000" patient_id="2" encounter_id="6"   creator="1" date_created="2012-01-01 00:00:00.0" voided="false" uuid="1c96f25c-4949-4f72-9931-d808fbcdb142" />
 
-</dataset> 
\ No newline at end of file
+
+	<order_group_attribute_type order_group_attribute_type_id="1" name="Stool" min_occurs="5" creator="1" date_created="2020-07-29 14:33:10.0" retired="1" uuid="9cf1b9de-d18e-11ea-87d0-0242ac130003"/>
+	<order_group_attribute_type order_group_attribute_type_id="2" name="Virology" min_occurs="5" creator="1" date_created="2020-07-29 14:33:10.0" retired="1" uuid="9cf1bbe6-d18e-11ea-87d0-0242ac130003"/>
+	<order_group_attribute_type order_group_attribute_type_id="3" name="Bacteriology" min_occurs="5" creator="1" date_created="2020-07-29 14:33:10.0" retired="1" uuid="9cf1bce0-d18e-11ea-87d0-0242ac130003"/>
+	<order_group_attribute_type order_group_attribute_type_id="4" name="ECG" min_occurs="5" creator="1" date_created="2020-07-29 14:33:10.0" retired="1" uuid="9cf1bdb2-d18e-11ea-87d0-0242ac130003"/>
+	
+<!--	<order_group_attribute order_group_attribute_id="1" order_group_id="1" attribute_type_id="1" value_reference="Test 1" uuid="86bdcc12-d18d-11ea-87d0-0242ac130003" creator="1" date_created="2020-07-29 14:33:10.0" voided="false"/>-->
+<!--	<order_group_attribute order_group_attribute_id="2" order_group_id="2" attribute_type_id="2" value_reference="Test 2" uuid="86bdcf82-d18d-11ea-87d0-0242ac130003" creator="1" date_created="2020-07-29 14:33:10.0" voided="false"/>-->
```

Comment:
Do you still need these commented out lines?

Reply:
Am Gon remove them .

Thanks alot for the feedback ..
Am going to clean out everything and add the remaining tests.